uber/tests/email/test_automated_email_job.py

```diff
@@ -0,0 +1,36 @@
+from uber.tests.email.email_fixtures import *
+
+
+@pytest.fixture
+def send_all_emails_mock():
+    with patch.object(SendAllAutomatedEmailsJob, '_send_all_emails', return_value=None) as mock:
+        yield mock
```

Comment:
Huh, apparently in the latest pytest you can just use ``yield`` from a normal ``@pytest.fixture`` without having to declare it as a ``@pytest.yield_fixture``.  Good stuff.,